.github/workflows/actions.yml

```diff
@@ -25,12 +25,14 @@ jobs:
       fail-fast: false
     steps:
       - name: "Checkout"
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
       - name: "Setup Julia"
         uses: julia-actions/setup-julia@v1
```

Comment:
The log file shows a warning message related to this line:
```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: julia-actions/setup-julia@v1, actions/cache@4d4ae6ae148a43d0fd1eda1800170683e9882738, pyTooling/Actions/with-post-step@adef08d3bdef092282614f3b683897cefae82ee3. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```
Is there a newer version of this script available?

Reply:
Looks like it's still at v1: https://github.com/julia-actions/setup-julia/releases/tag/v1.9.5